bhohnco/kingsCastle
# Feature/add favorite button

Is this a fix or a feature?
* feature
What is the change or fix?
* favorite button is added to cards 
* needs to be adjusted to toggle color on click 
How should this be tested?
* load the page and check for the favorite button 
Linked Issues: #12 #2 #11 
Add A Reviewer if Necessary
*
